sidecarDockerrun.sh

```diff
@@ -27,10 +27,17 @@ sed -i "s/LogFilePath: \"fuse_log.txt\"/LogFilePath: \"\/data\/_manifest-sync-st
 trap cleanup SIGTERM
 
 WTS_STATUS=$(curl -s -o /dev/null -I -w "%{http_code}" http://workspace-token-service.$NAMESPACE/_status)
-if [[ ( "$WTS_STATUS" -ne 200 ) ]]; then
-    echo "Unable to reach WTS at 'http://workspace-token-service.$NAMESPACE', or WTS is not healthy"
-    exit 1
-fi
+WTS_RETRY_COUNTER = 0
+while [[ ( "$WTS_STATUS" -ne 200 ) ]]; do
+    if [[ ( "$WTS_RETRY_COUNTER" -ge 20 ) ]]; then
+        "Still unable to reach WTS at 'http://workspace-token-service.$NAMESPACE', or WTS is not healthy after 5 minutes. Check WTS status."
+        exit 1
```

Comment:
actually I'm a bit unsure if wee need this

fuse definately needs wts, but what if a workspace pod doesn't need fuse? the fuse sidecar should not prevent the workspace pod from starting up maybe?

Reply:
sure we could just keep the loop running then?

Reply:
yeah that is my thought, both have its pros and cons